shotover-proxy/tests/scripts/cassandra-cluster.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
I dont think we need this.
Running `docker-compose -f docker-compose.yml up` is already pretty simple.
I'm not sure what the defer is for, docker-compose already runs down when the up is killed.

Reply:
Oh I see!
Fair enough